spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/RoutingKeyGenerator.scala

```diff
@@ -7,7 +7,7 @@ import com.datastax.spark.connector.cql.TableDef
 
 /** This class computes the routing key of a bound statement. */
 class RoutingKeyGenerator(table: TableDef, columnNames: Seq[String])
-  extends ((BoundStatement) => ByteBuffer) {
+  extends ((BoundStatement) ⇒ ByteBuffer) {
```

Comment:
Did you add scala formatting? Unclear why you added ⇒ vs =>, ⇒ is what formatting adds